packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/optimize-ciemss-operation.ts

```diff
@@ -60,6 +60,8 @@ export interface OptimizeCiemssOperationState extends BaseState {
 	constraintGroups: ConstraintGroup[];
 	// Charts + Outputs:
 	chartConfigs: string[][];
+	selectedInterventionCharts: string[];
+	selectedSimulationCharts: string[];
```

Comment:
A bit of cleanup and clarification would be appreciated here
- it seems like you are not selecting charts, but rather variables
- fyi it seems "selectedInterventionCharts" is not used
- if you don't use chartConfigs anymore, consider removing it